ros_buildfarm/sourcerpm_job.py

```diff
@@ -55,6 +55,22 @@ def build_sourcerpm(
         '--depth', '1', '--no-single-branch',
         release_repository_url, os_pkg_name]
 
+    # HACK FOR RHBZ#1960791
```

Comment:
Is there a reason this can't or shouldn't be a URL to the bug so that anyone can follow it?